libyul/backends/wasm/EVMToEWasmTranslator.cpp

```diff
@@ -341,6 +771,12 @@ function u256_to_i32(x1, x2, x3, x4) -> v {
 	v := x4
 }
 
+function u256_to_byte(x1, x2, x3, x4) -> v {
+	if i64.ne(0, i64.or(i64.or(x1, x2), x3)) { invalid() }
+	if i64.ne(0, i64.shr_u(x4, 56)) { invalid() }
```

Comment:
```suggestion
	if i64.ne(0, i64.shr_u(x4, 8)) { invalid() }
```
?
Maybe it is better to check if `x4` is larger than 255?

Reply:
Agreed. Will change to i64.gt_u(x4, 255)